wagtail/utils/timestamps.py

```diff
@@ -0,0 +1,41 @@
+from django.conf import settings
+from django.utils import formats, timezone
+from django.utils.dateparse import parse_datetime
+
+
+def ensure_utc(value):
+    """
+    Similar to how django-modelcluster stores the revision's data and similar to how
+    django stores dates in the database, this converts the date to UTC if required.
+    """
+    # https://github.com/wagtail/django-modelcluster/blob/8666f16eaf23ca98afc160b0a4729864411c0563/modelcluster/models.py#L21-L28
+    if settings.USE_TZ:
+        if timezone.is_naive(value):
+            default_timezone = timezone.get_default_timezone()
+            value = timezone.make_aware(value, default_timezone).astimezone(
+                timezone.utc
+            )
+        else:
```

Comment:
The code is based on django-modelcluster, but think I've found a small improvement compared to the modelcluster's implementation. So I have created a PR there: https://github.com/wagtail/django-modelcluster/pull/168